spacy_experimental/cli/benchmark.py

```diff
@@ -0,0 +1,94 @@
+from pathlib import Path
+from typing import Iterable
+from spacy import Language, util
+from spacy.cli import app
+from spacy.tokens import Doc
+from spacy.training import Corpus
+from thinc.api import require_gpu
+from thinc.util import gpu_is_available
+import time
+from typer import Argument as Arg, Option
+from wasabi import Printer
+
+
+@app.command("benchmark")
+def benchmark_cli(
+    model: str = Arg(..., help="Model name or path"),
+    data_path: Path = Arg(
+        ..., help="Location of binary evaluation data in .spacy format", exists=True
+    ),
+    use_gpu: int = Option(-1, "--gpu-id", "-g", help="GPU ID or -1 for CPU"),
+    warmup_epochs: int = Option(
+        3, "--warmup", "-w", min=0, help="Number of iterations over the data for warmup"
+    ),
+    bench_epochs: int = Option(
+        3,
+        "--iter",
+        "-i",
+        min=1,
+        help="Number of iterations over the data for benchmarking",
+    ),
+):
+    """
+    Benchmark a pipeline. Expects a loadable spaCy pipeline and benchmark
+    data in the binary .spacy format.
+    """
+    setup_gpu(use_gpu=use_gpu, silent=False)
```

Comment:
Don't we want to expose `silent` as an command line arg?

Reply:
Mimicking the behavior of `spacy evaluate` here.